tests/Common/SpanIntegrationChecker.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace DDTrace\Tests\Common;
+
+use DDTrace\Contracts\Span;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Checks that spans are assigned the proper integration.
+ */
+class SpanIntegrationChecker
```

Comment:
Probably doesn't matter, but just to lock this down a bit. :)

```suggestion
final class SpanIntegrationChecker
```